internal/dag/ingress_processor.go

```diff
@@ -95,7 +95,7 @@ func (p *IngressProcessor) computeSecureVirtualhosts() {
 
 func (p *IngressProcessor) computeIngresses() {
 	// deconstruct each ingress into routes and virtualhost entries
-	for _, ing := range p.source.ingresses {
+	for _, ing := range p.source.ingressesv1beta1 {
```

Comment:
I think we're going to need to check both `kc.ingresses` and `kc.ingressesv1beta1`, since either one of them could be populated based on whether `v1` exists in the cluster or not.